<launch>
<rosparam command="load" file="$(find Throwing_ball)/config/sensor_data_config.yaml" />
    <node pkg="Throwing_ball"
          name="sensor_data"
          type="sensor_data"
          output="screen"/>
    <node pkg="Throwing_ball"
          name="kalman_filter"
          type="kalman_filter"
          output="screen"/>
</launch>